roles/agent/meta/main.yml

```diff
@@ -28,9 +28,9 @@ galaxy_info:
         - '7'
         - '8'
         - '9'
-    - name: Amazon Linux 2
+    - name: Amazon Linux
       versions:
-        - all
```

Comment:
why not keep `all` for Amazon Linux since we dropped AL2?

Reply:
keeping all would mean we support versions 1, 2 and 2023: https://github.com/ansible/ansible-lint/blob/main/src/ansiblelint/schemas/meta.json#L51